clinica/pipelines/statistics_volume/template_model_creation.m

```diff
@@ -0,0 +1,17 @@
+spm('defaults','pet');
+spm_jobman('initcfg');
+
+matlabbatch{1}.spm.stats.factorial_design.dir = {@OUTPUTDIR};
+matlabbatch{1}.spm.stats.factorial_design.des.t2.scans1 = {@SCANS1}';
+matlabbatch{1}.spm.stats.factorial_design.des.t2.scans2 = {@SCANS2}';
+matlabbatch{1}.spm.stats.factorial_design.des.t2.dept = 0;
+matlabbatch{1}.spm.stats.factorial_design.des.t2.variance = 1;
+matlabbatch{1}.spm.stats.factorial_design.des.t2.gmsca = 0;
+matlabbatch{1}.spm.stats.factorial_design.des.t2.ancova = 0;
+matlabbatch{1}.spm.stats.factorial_design.multi_cov = struct('files', {}, 'iCFI', {}, 'iCC', {});
+matlabbatch{1}.spm.stats.factorial_design.masking.tm.tm_none = 1;
+matlabbatch{1}.spm.stats.factorial_design.masking.im = 1;
+matlabbatch{1}.spm.stats.factorial_design.masking.em = {''};
+matlabbatch{1}.spm.stats.factorial_design.globalc.g_omit = 1;
+matlabbatch{1}.spm.stats.factorial_design.globalm.gmsca.gmsca_no = 1;
+matlabbatch{1}.spm.stats.factorial_design.globalm.glonorm = 1;
```

Comment:
Why this matlab file is the only one which does not have `spm_jobman('run', matlabbatch)`?

Reply:
Because `model_creation()` does a bunch of thing (adding all the needed covariates), and at the end writes 
```
with open(current_model, 'a') as file:
    file.write('spm_jobman(\'run\', matlabbatch)')
```